core/keys/arrow_key.talon-list

```diff
@@ -0,0 +1,6 @@
+list: user.arrow_key
+-
+down: down
+left: left
+right: right
+up: up
```

Comment:
You can just do `up` instead of `up: up` in all of these identity mapped lists. With up: up it's actually kind of confusing which side to edit too.

Reply:
It's arguably a bit more convenient for those that may customize to keep up: up though.

This is a similar problem to eg https://github.com/talonhub/community/blob/7b09a4a789ed1dc380865d2840a5abda9f65af25/core/windows_and_tabs/window_snap_positions.talon-list

where we may be better off with something more obviously not a spoken form on the right side.

Reply:
I think it's not more convenient, because it's not obvious which side to edit when the sides are symmetrical.